games/static/games/proto.js

```diff
@@ -325,10 +326,15 @@ class Board {
             this.updateIndexOf(rocket)
             rocketLocation = this.getIndexOf(rocket)
             nextRocketLocation = this.update(nextRocketLocation)
+            // The next if statement checks if the rocket is at the top of the board
+            // and if so erases it after 100 Milliseconds
             if (this.boardTopIs(rocketLocation)) {
                 clearInterval(rocketId)
                 this.waitAndEraseRocketAt(rocketLocation) 
             }
+            // if there is a next square remove dead alien on it if there is one
+            // and if it is an alien erase the rocket, reduce the aliens health by
+            // one and set the square's css according to the new alien's health level
             if (this.squareAt(nextRocketLocation)) {
```

Comment:
i see this a lot:
`this.squareAt(nextRocketLocation)`

please extract it to its own variable:
```js
let square = this.squareAt(nextRocketLocation)
if (square) { ... }
```

Reply:
this made a lot of problems, let's keep it for that specific PR we are reserving for this function